tests/dag_block_test.cpp

```diff
@@ -194,29 +193,6 @@ TEST_F(DagBlockTest, push_and_pop) {
   blk_qu.stop();
 }
 
-TEST_F(DagBlockTest, overlap) {
```

Comment:
Why is this test removed ?

Reply:
As you wrote in #928 (comment) TransactionOrderManager wasn’t used anywhere, but that test. So I removed it and that related test. I don't think we need to test unused functionality

Reply:
true